packages/vitest/src/node/config.ts

```diff
@@ -69,6 +69,9 @@ export function resolveConfig(
   if (resolved.api === true)
     resolved.api = defaultPort
 
+  if (options.host)
```

Comment:
we may also configure vite configuration when missing from vite and provided from vitest...